django/db/backends/base/schema.py

```diff
@@ -1045,7 +1045,11 @@ def create_fk_name(*args, **kwargs):
     def _delete_fk_sql(self, model, name):
         return self._delete_constraint_sql(self.sql_delete_fk, model, name)
 
-    def _unique_sql(self, model, fields, name, condition=None):
+    def _unique_sql(self, model, fields, name, condition=None, defer=None):
+        from django.db.models.constraints import NOT_DEFERRABLE
+        if defer is None:
+            defer = NOT_DEFERRABLE
+
```

Comment:
IMO we don't need a `NOT_DEFERRABLE` constant. I would remove it and use `''` here and `None` in `UniqueConstraint.__init__()`), e.g.
```python
constraint = self.sql_unique_constraint % {
    'columns': ', '.join(map(self.quote_name, fields)),
    'defer': defer or '',
}
```
```python
class UniqueConstraint(BaseConstraint): 
    def __init__(self, *, fields, name, condition=None, defer=None)
```